code/datums/spells/lordspells.dm

```diff
@@ -0,0 +1,80 @@
+// Necrolord
+/obj/effect/proc_holder/spell/targeted/trigger/soulflare
+	name = "Soulflare"
+	desc = "Deals high damage to an enemy in 3 different damage types, as well as paralyzing them for 5 seconds. If it hits an enemy in critical condition, it instantly kills them."
+	school = "transmutation"
+	charge_max = 300
+	clothes_req = 1
+	invocation = "NEKROSIS"
+	invocation_type = "shout"
+	message = "<span class='notice'>Your head feels like it's being burned as you fall to the ground!</span>"
+	cooldown_min = 50 //12 deciseconds reduction per rank
+
+	starting_spells = list("/obj/effect/proc_holder/spell/targeted/inflict_handler/soulflare")
+
+	action_icon_state = "soulflare"
+
+/obj/effect/proc_holder/spell/targeted/inflict_handler/soulflare
+	amt_paralysis = 5
+	amt_dam_fire = 15
+	amt_dam_brute = 15
+	amt_dam_tox = 15
+	sound="sound/magic/Necrolord_Soulflare_Cast.ogg"
+
+/obj/effect/proc_holder/spell/targeted/inflict_handler/soulflare/cast(list/targets, mob/user = usr)
+	var/obj/effect/proc_holder/spell/targeted/trigger/soulflare/SF = locate(/obj/effect/proc_holder/spell/targeted/trigger/soulflare, user.mob_spell_list)
+	var/mob/living/carbon/target = targets[1]
```

Comment:
Won't this just select the top name of the list rather than allowing you to choose your target? Shouldn't this work like curse of the barnyard or something?